creator-node/src/services/stateMachineManager/stateReconciliation/stateReconciliationUtils.js

```diff
@@ -101,6 +101,14 @@ const getNewOrExistingSyncReq = async ({
   return { syncReqToEnqueue }
 }
 
+const getSyncStatusByUuid = async (targetUrl, syncUuid) => {
+  const resp = await axios({
+    url: `${targetUrl}/sync_status/uuid/${syncUuid}`,
+    method: 'get'
+  })
+  return resp?.data?.data?.syncStatus
+}
```

Comment:
should have timeout / other protection on this route
longer term using raw axios is bad bc it amplifies bad practices
prob a wrapper that forces you to set a timeout would be good
obv don't do

^i dub this the axios axiom

Reply:
good callout for the axios axiom haha

we have a couple other raw axios calls in this function but for now I just added a timeout of 30sec to this, which is generous. also fixed to use baseURL + url instead of just url

Reply:
cool